pandas/io/tests/test_pytables.py

```diff
@@ -2665,23 +2665,6 @@ def test_sparse_frame(self):
         self._check_double_roundtrip(ss3, tm.assert_frame_equal,
                                      check_frame_type=True)
 
-    def test_sparse_panel(self):
```

Comment:
Well, unless specific files can be pointed out, a GitHub search is sufficient to convince me that they are not there.